include/authz.php

```diff
@@ -61,26 +62,58 @@ function setUsername() {
 	// Private function to simplify creation of common SVN authz command string text.
 	function svnAuthzCommandString($repos, $path, $checkSubDirs = false) {
 		global $config;
-		return $config->getSvnAuthzCommand().' --repository '.$repos.' --path '.quote($path).
-			' '.($this->hasUsername() ? '--username '.quote($this->user).' ' : '').($checkSubDirs ? '-R ' : '').quote($this->accessfile);
+
+		$cmd			= $config->getSvnAuthzCommand();
+		$repoAndPath	= '--repository ' . quote($repo) . ' --path ' . quote($path);
```

Comment:
Here is a severe issue, has to be `$repos`

Reply:
It has to be `$repo` in the function signature instead, must have C&Ped something instead of cherry-picked.